homeassistant/components/sia/__init__.py

```diff
@@ -0,0 +1,51 @@
+"""The sia integration."""
+import asyncio
+
+from homeassistant.components.alarm_control_panel import (
+    DOMAIN as ALARM_CONTROL_PANEL_DOMAIN,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+from .hub import SIAHub
+
+PLATFORMS = [ALARM_CONTROL_PANEL_DOMAIN]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the sia component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up sia from a config entry."""
+    hub = SIAHub(hass, entry.data, entry.entry_id, entry.title)
+
```

Comment:
```suggestion

async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
    """Set up sia from a config entry."""
    hub = SIAHub(hass, entry.data, entry.entry_id, entry.title)
    hass.data.setdefault(DOMAIN, {})

```

You can remove `async_setup` once this is rebased to `upstream/dev`